packages/storybook/src/stories/playground/customize-components.stories.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { ColDef, XGrid, GridOverlay, Footer, useApiRef, ApiRef } from '@material-ui/x-grid';
+import { ColDef, XGrid, GridOverlay, GridFooter, useApiRef, ApiRef } from '@material-ui/x-grid';
```

Comment:
Prefix public components with `Grid`, as we will re-export them from `@material-ui/x` later on.